IDEAS/Examples/Tutorial/DetailedHouse/DetailedHouse5.mo

```diff
@@ -1,5 +1,5 @@
-within IDEAS.Examples.Tutorial;
-model Example5 "New model with 2 coupled zones"
+within IDEAS.Examples.Tutorial.DetailedHouse;
+model DetailedHouse5 "New model with 2 coupled zones"
   extends Modelica.Icons.Example;
   replaceable package Medium = IDEAS.Media.Air "Air medium";
 
```

Comment:
- Change zone height to 2.8m
- Remove obsolete whitespace at line 10 (sim)
- Rename instances, e.g. `zon1` and `zon2` or `recZon1` and `recZon2`